action.yml

```diff
@@ -13,3 +16,41 @@ runs:
       run: ${{ github.action_path }}/scripts/main.sh
       env:
         YARN_NPM_AUTH_TOKEN: ${{ inputs.npm-token }}
+    - id: name-version
+      shell: bash
+      if: inputs.slack-webhook-url != ''
+      run: |
+        NAME_VERSION=$(jq --raw-output '.name + "@" + .version' package.json)
+        echo "NAME_VERSION=$NAME_VERSION" >> "$GITHUB_OUTPUT"
+    - name: Post to a Slack channel
+      id: slack
+      if: inputs.slack-webhook-url != ''
+      uses: slackapi/slack-github-action@v1.22.0
+      with:
+        payload: |
+          {
+            "text": "`${{ steps.name-version.outputs.NAME_VERSION }}` is awaiting review :rocket: : https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}/",
+            "blocks": [
+              {
+                "type": "section",
+                "text": {
+                  "type": "mrkdwn",
+                  "text": "`${{ steps.name-version.outputs.NAME_VERSION }}` is awaiting review :rocket:"
+                },
+                "accessory": {
+                  "type": "button",
+                  "text": {
+                    "type": "plain_text",
+                    "text": "Review",
+                    "emoji": true
+                  },
+                  "value": "review_deploy",
```

Comment:
It seems that we don't need this value (see https://api.slack.com/reference/block-kit/block-elements#button)

Though it is included in the "interaction payload" sent to the "app" this is configured with. How do you have that setup now?

Presumably there is some sort of "Slack App" that this needs to be configured with, and it will handle these interactions by opening the URL in a tab. We should document any such requirements here.

Reply:
Interesting!

So upon clicking the review button I did see a little warning:

![image](https://user-images.githubusercontent.com/675259/203662493-907e9438-ed0c-4ea6-84e2-43f47f06d32f.png)

I just figured this was something we could configure when we set it up, but I should see what happens when it's removed. Good catch!

Reply:
I think it's something you need to setup in the app either way. This value is just sent along with that payload.

Reply:
Maybe we should just use a regular link instead?

Reply:
I've gone ahead and make this a markdown link instead: 
![image](https://user-images.githubusercontent.com/675259/203841639-da2d49f4-62fb-4773-93c5-eea77d5c87b3.png)

Reply:
Looks good!